USER-GUIDE.md

```diff
@@ -0,0 +1,484 @@
+# KCP-Edge User Guide:
+
+## Required Packages:
+   - docker
+   - kind
```

Comment:
kind is also versioned.  What is important here is the kubernetes version that kind will create.

Reply:
I don't understand this. comment - could you clarify? we are already providing a range for the acceptable k8s version

Reply:
If I recall correctly, installing a particular version of kind implies the version of Kubernetes that kind will create by default.  If that is right then we should say here what versions of kind will create usable versions of Kubernetes by default.